scripts/arangodb-bash-setup/js/db_setup.js

```diff
@@ -0,0 +1,33 @@
+const users = require('@arangodb/users');
+
+print("Running DB Setup as root in Arangosh\n");
+
+print("Drop database: ", process.env.ARANGO_DB_NAME);
+if (db._dropDatabase(process.env.ARANGO_DB_NAME)) {
+  print("Database dropped successfully\n");
+}
+
+print("Create database: ", process.env.ARANGO_DB_NAME);
+if (db._createDatabase(process.env.ARANGO_DB_NAME)) {
+  print("Database created successfully\n");
+}
+
+const userResult = users.document(process.env.ARANGO_DB_USER);
+
+if (userResult.code !== 200) {
+  print(`User: ${process.env.ARANGO_DB_USER} not found`);
+}
+else {
```

Comment:
Normally I'd wrap this code in a function and return early to avoid too many if \ else blocks. But I think in this context, it's fine.